tasks/information/hiddenhandApi.ts

```diff
@@ -51,10 +51,13 @@ export const validateDeadLine = (proposals: Array<Proposal>): boolean => {
     const now = Number.parseInt((new Date().getTime() / 1000).toFixed(0));
     return proposals[0].proposalDeadline > now;
 };
-export const isAuraBalProposal = (p: Proposal) => p.proposal == 1 && p.title == "Stable auraBAL/B-80BAL-20WETH";
-export const isAuraEthProposal = (p: Proposal) => p.proposal == 2 && p.title == "50/50 AURA/WETH";
+export const isAuraBalProposal = (p: Proposal) => p.title == "Stable auraBAL/B-80BAL-20WETH"; // proposal 0x0312aa8d0ba4a1969fddb382235870bf55f7f242
```

Comment:
Leaving those comments to review if with v2 the proposal keeps steady or changes